guide/inflation.rst

```diff
@@ -2,146 +2,246 @@
-in the filtering problem. Prior inflation is able to address issues in the forecast 
-step such as model errors while posterior inflation can help mitigate sampling errors 
-in the analysis step. 
-
-
-Inflation values can vary in space and time, depending on the specified namelist values. Even though we talk about a
-single inflation value, the inflation has a probability density with a mean and standard deviation. We use the mean
-value when we inflate, and the standard deviation indicates how sure of the value we are. Larger standard deviation
-values mean "less sure" and the inflation value can increase more quickly with time. Smaller values mean "more sure" and
-the time evolution will be slower since we are more confident that the mean (inflation value) is correct.
-
-The standard deviation of inflation allows inflation values to increase with time, if required by increasing density or
-frequency of observations, but it does not provide a mechanism to reduce the inflation when the frequency or density of
-observations declines. So there is also an option to damp inflation through time. In practice with large geophysical
-models using damped inflation has been a successful strategy.
-
-The following namelist items which control inflation are found in the ``input.nml`` file, in the &filter_nml namelist.
-The detailed descriptions are in the `filter_mod <../assimilation_code/modules/assimilation/filter_mod.html#Namelist>`__ page. Here we
-try to give some basic advice about commonly used values and suggestions for where to start. Spatial variation is
-controlled by ``inf_flavor``, which also controls whether there's any inflation, ``inf_initial_from_restart``, and
-``inf_initial``, as described below. Time variation is controlled by ``inf_sd_initial_from_restart``,
-``inf_sd_initial``, ``inf_sd_lower_bound``, ``inf_damping``, ``inf_lower_bound`` and ``inf_upper_bound``.
-
-In the namelist each entry has two values. The first is for Prior inflation and the second is for Posterior inflation.
+Sampling errors and model errors are significant challenges that degrade the quality of the sample
+background covariance matrix in the ensemble Kalman filter (EnKF). Sampling errors typically arise
+from the use of a limited-size ensemble due to computational constraints. Conversely, model errors
+and biases stem from deficiencies in model dynamics and uncertainties in physical parameters.
+These issues can result in an underestimation of the true (and unknown) variance of the system.
+
+Inflation is a common technique used to restore variability in the ensemble without altering its mean.
+This is achieved by linearly scaling the ensemble perturbations (also known as anomalies or
+departures from the mean) as follows:
+
+	:math:`\widetilde{\mathbf{x}}_k^i = \sqrt{\lambda} \left( \mathbf{x}_k^i -
+ 	\overline{\mathbf{x}}_k \right) + \overline{\mathbf{x}}_k, \quad
+	i = 1, 2, ..., N_e`
+
```

Comment:
Does the subscript 'k' serve any purpose here?  It's not defined and I wasn't sure if 'k' was the the same thing as 'e' the total ensemble members.